spec/timezone.html

```diff
@@ -332,7 +324,7 @@ <h1>Temporal.TimeZone.prototype.getPreviousTransition ( _startingPoint_ )</h1>
     <emu-clause id="sec-temporal.timezone.prototype.tostring">
       <h1>Temporal.TimeZone.prototype.toString ( )</h1>
       <p>
-        The following steps are taken:
+        The `Temporal.TimeZone.prototype.toString` function performs the following steps when called:
```

Comment:
Is this not a method rather than a function, if we wanna be pedantic?

Reply:
To be pedantic, nothing’s really a method, they’re all just function-valued properties :-) but typically yes we’d describe receiver-sensitive functions as a method.

However, the spec usually just says “the following steps are performed” - why do we need to repeat the name of the function/method here?

Reply:
That's what is asked for in https://github.com/tc39/ecma262/issues/2304.

Reply:
I only see one agreed-on option there? It has two variations, one without any additional informative description, and one with.